src/main/java/com/marklogic/kafka/connect/DefaultDatabaseClientConfigBuilder.java

```diff
@@ -17,7 +43,28 @@ public DatabaseClientConfig buildDatabaseClientConfig(Map<String, String> kafkaC
 		DatabaseClientConfig clientConfig = new DatabaseClientConfig();
 		clientConfig.setCertFile(kafkaConfig.get(MarkLogicSinkConfig.CONNECTION_CERT_FILE));
 		clientConfig.setCertPassword(kafkaConfig.get(MarkLogicSinkConfig.CONNECTION_CERT_PASSWORD));
-
+		clientConfig.setTrustManager(new SimpleX509TrustManager());
+		
+		/*
+		 * v1.2.2 changes START
+		 */
+		String ssl = kafkaConfig.get(MarkLogicSinkConfig.SSL);
```

Comment:
I recommend moving this into a private method to encapsulate the logic here - e.g. "configureHostnameVerifier". I also would avoid comments like this - if someone wants to know when code was added, they can use their version history tool to figure that out.

Reply:
Will make the changes